Ghidra/Processors/V850/data/languages/V850.cspec

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<compiler_spec>
+
+  <global>
+    <range space="ram"/>
+  </global>
+
+  <stackpointer register="sp" space="ram"/>
+
+  <default_proto>
+    <prototype name="__stdcall" extrapop="0" stackshift="0">
+      
+      <input>
+        <pentry minsize="1" maxsize="4">
+          <register name="r6"/>
+        </pentry>
+
+        <pentry minsize="1" maxsize="4">
+          <register name="r7"/>
+        </pentry>
+
+        <pentry minsize="1" maxsize="4">
+          <register name="r8"/>
+        </pentry>
+
+        <pentry minsize="1" maxsize="4">
+          <register name="r9"/>
+        </pentry>
+
+        <pentry minsize="1" maxsize="500" align="4">
+          <addr offset="0" space="stack"/>
+        </pentry>
+      </input>
+      
+      <output>
+        <pentry minsize="1" maxsize="4">
+          <register name="r10"/>
+        </pentry>
+      </output>
```

Comment:
I did some small tests with `pcodetest` utility. GCC seems to be able to handle 64-values. Please add 
```
        <pentry minsize="5" maxsize="8">
          <addr space="join" piece1="r10" piece2="r11"/>
        </pentry>
```
in `output` section.
Unfortunately, I didnt know how to correctly specify this for input values.